src/client/containers/PullRequestDiff/__tests__/index.js

```diff
@@ -0,0 +1,69 @@
+import chai from 'chai'
+import {
+  getData,
+} from '../index'
+import { types } from 'ducks/pullrequests/actions'
+
+const expect = chai.expect
+
+describe('getData', () => {
+  it('full data', () => {
```

Comment:
>When can it occur that there would be no status?
It can occur for many reasons, someone broke fetch logic, someone renamed actions types, someone changes redux state structure, etc